test/default.cnf

```diff
@@ -8,6 +8,10 @@ providers = provider_sect
 
 [provider_sect]
 default = default_sect
+legacy  = legacy_sect
 
 [default_sect]
-activate = 1
+activate = true
+
+[legacy_sect]
+activate = false
```

Comment:
Is this config file used in a test that somehow verifies that the legacy provider is not loaded?

Reply:
dang, I meant to add a test that specifically checked for the legacy provider not being loaded, and must have missed it.  I'll add it